cve_bin_tool/checkers/docker.py

```diff
@@ -0,0 +1,20 @@
+"""
+CVE checker for docker
+https://www.cvedetails.com/product/28125/Docker-Docker.html?vendor_id=13534
+"""
+
+from __future__ import annotations
+
+from cve_bin_tool.checkers import Checker
+
+
+class DockerChecker(Checker):
+    CONTAINS_PATTERNS: list[str] = []
+    FILENAME_PATTERNS: list[str] = [r"docker"]
+    VERSION_PATTERNS: list[str] = [
+        r"docker-ce-([0-9]+\.[0-9]+\.[0-9]+)",
+        r"moby-([0-9]+\.[0-9]+\.[0-9]+)",
```

Comment:
Now that you've made a separate moby checker, do you really want this pattern here?

Reply:
Yes, it'll be necessary for `docker-24.0.7-2-omv2390.aarch64.rpm` to detect docker. 
However, this checker won't be able to detect moby that'll be done by moby checker.